src/ride_type.cpp

```diff
@@ -631,6 +648,8 @@ void RidesManager::NewInstanceAdded(uint16 num)
 	const RideType *rt = ri->GetRideType();
 	assert(ri->state == RIS_ALLOCATED);
 
+	ri->InsertIntoWorld();
+
```

Comment:
I think we should discuss empty lines.

You seem to like double-spaced code (one line code one line empty) for some reason, but it throws away the value of an empty line.

In the program, code that perform a 'step' (a closely releated coherent set of actions towards the final goal) has no empty lines. There is an empty between steps. Often a step begins with a comment line explaining what the step is aiming to achieve (this is not a rule, but it's useful for browsing code).

What the above does is making visually separated blocks of code that belong together. Especially with comment lines above blocks, you can simply skip blocks of code by just reading the comment line, until you found the block that tries to do what you're looking for. Only at that point you have to start reading actual code.

If you make single line "blocks" this system breaks, you still have to read each line of code, at which point it's better to simply delete all empty lines, as it shrinks the function by 50% lines.

So please avoid making 1-line blocks, probably by making the 'step' that a block performs bigger or at a higher abstraction level.

In this case, adding the new line to the block above seems a good solution.

Reply:
Okay. I'm still getting accustomed to the code style used in this project, but I'm confident such mistakes will get less frequent quickly.